src/streamlink/plugin/plugin.py

```diff
@@ -189,6 +190,10 @@ def set_option(cls, key, value):
     def get_option(cls, key):
         return cls.options.get(key)
 
+    @classmethod
+    def get_argument(cls, key):
```

Comment:
`self.arguments.get` and `self.get_argument` are
`<streamlink.options.Argument object at 0x0000000>`

shouldn't it be like `self.options.get` and `self.get_options`

Reply:
I did intend it to return the actual `Argument` object, in this case `get_option` should still be used. And as you pointed out in the other comment, the name of the option/argument should be normalised :)